tests/test_wfirst.py

```diff
@@ -536,15 +536,24 @@ def test_wfirst_detectors():
     im_list = [im0_1,im0_2]*4
 
     im_1.applyPersistence(im_list,galsim.wfirst.persistence_coefficients)
-    galsim.wfirst.applyPersistence(im_2, im_list)
+    galsim.wfirst.applyPersistence(im_2, im_list, method='linear') #check the linear method
     assert im_2.scale == im_1.scale
     assert im_2.wcs == im_1.wcs
     assert im_2.dtype == im_1.dtype
     assert im_2.bounds == im_1.bounds
     np.testing.assert_array_equal(
-        im_2.array, im_1.array,
-        err_msg='Persistence results depend on function used.')
+       im_2.array, im_1.array,
+       err_msg='Persistence results depend on function used.')
+
+    im_unit = galsim.Image(np.ones((2,2)), copy=True)
+    im_f = im_unit*0.0
+    im_f_list = [im_unit*1.E3, im_unit*1.E5]
+    galsim.wfirst.applyPersistence(im_f, im_f_list, method='fermi') #check fermi method
+    #Check the functionality of the fermi method by comparing with pre-calculated values.
+    assert np.allclose( im_f.array, np.ones((2,2))*8.30515, rtol=1.E-06 ), 'Error in Fermi persistence model'
```

Comment:
Explain where this 8.30515 value comes from.

Reply:
The value is pre-calculated using my local function that reproduces the reference curve provided by Chris. 
![avg_persistence_602](https://user-images.githubusercontent.com/39343677/58375375-3ddb3280-7f1f-11e9-8097-f517c8f3ea90.png)
![persistence_model](https://user-images.githubusercontent.com/39343677/58375378-46336d80-7f1f-11e9-9cf1-b2e0c8001eec.png)

This value is updated since we changed the illumination level of previous exposures in the test.
Here's the log that shows how the this value is computed:

exposure_id /  mid-time /  illumination  / persistence 
     0        70.125       1e+03       3.04e-01  
     1       210.375       1e+04       1.01e+00  
     2       350.625       4e+04       2.43e+00  
     3       490.875       5e+04       2.16e+00  
     4       631.125       5e+04       1.69e+00  
     5       771.375       1e+05       2.18e+00  
     6       911.625       1e+06       2.97e+00  
total persistence 12.748214217103115